Podcast/RecommendedCard.swift

```diff
@@ -7,23 +7,55 @@
     
-    init(episodeID: Int, episodeTitle: String = "", dateCreated: Date = Date(), descriptionText: String = "Not avaliable", smallArtworkImageURL: URL , time: Double = 0.0, numberOfRecommendations: Int = 0, tags: [Tag] = [], seriesTitle: String = "", seriesID: Int = 0, isBookmarked: Bool = false, isRecommended: Bool = false, namesOfRecommenders: [String] = [], imagesOfRecommenders: [UIImage] = [], numberOfRecommenders: Int = 0) {
+    //initializer all atributes
+    init(episodeID: Int, episodeTitle: String = "", dateCreated: Date = Date(), descriptionText: String = "Not avaliable", smallArtworkImageURL: URL, episodeLength: Double = 0.0, numberOfRecommendations: Int = 0, tags: [Tag] = [], seriesTitle: String = "", seriesID: Int = 0, isBookmarked: Bool = false, isRecommended: Bool = false, namesOfRecommenders: [String] = [], imageURLsOfRecommenders: [URL] = [], numberOfRecommenders: Int = 0) {
         
         self.namesOfRecommenders = namesOfRecommenders
-        self.imagesOfRecommenders = imagesOfRecommenders
+        self.imageURLsOfRecommenders = imageURLsOfRecommenders
         self.numberOfRecommenders = numberOfRecommenders
         
-        super.init(episodeID: episodeID, episodeTitle: episodeTitle, dateCreated: dateCreated, descriptionText: descriptionText, smallArtworkImageURL: smallArtworkImageURL, episodeLength: time, numberOfRecommendations: numberOfRecommendations, tags: tags, seriesTitle: seriesTitle, isBookmarked: isBookmarked, isRecommended: isRecommended)
+        super.init(episodeID: episodeID, episodeTitle: episodeTitle, dateCreated: dateCreated, descriptionText: descriptionText, smallArtworkImageURL: smallArtworkImageURL, episodeLength: episodeLength, numberOfRecommendations: numberOfRecommendations, tags: tags, seriesTitle: seriesTitle, isBookmarked: isBookmarked, isRecommended: isRecommended)
     }
     
+    //initializer without smallArtworkImageURL options
+    init(episodeID: Int, episodeTitle: String = "", dateCreated: Date = Date(), descriptionText: String = "Not avaliable", episodeLength: Double = 0.0, numberOfRecommendations: Int = 0, tags: [Tag] = [], seriesTitle: String = "", seriesID: Int = 0, isBookmarked: Bool = false, isRecommended: Bool = false, namesOfRecommenders: [String] = [], imageURLsOfRecommenders: [URL] = [], numberOfRecommenders: Int = 0) {
+        
+        self.namesOfRecommenders = namesOfRecommenders
+        self.imageURLsOfRecommenders = imageURLsOfRecommenders
+        self.numberOfRecommenders = numberOfRecommenders
+        
+        super.init(episodeID: episodeID, episodeTitle: episodeTitle, dateCreated: dateCreated, descriptionText: descriptionText, episodeLength: episodeLength, numberOfRecommendations: numberOfRecommendations, tags: tags, seriesTitle: seriesTitle, isBookmarked: isBookmarked, isRecommended: isRecommended)
+    }
+    
+    
+    convenience init(json: JSON) {
+        let episodeID = json["episode_id"].int ?? 0
```

Comment:
for this one and everywhere else you do this, using intValue instead of int, stringValue instead of string, etc. gives a default value if its nil so you don't have to do the ?? nonsense